src/components/events/register/Form/RegistrationForm.tsx

```diff
@@ -57,7 +58,7 @@ const RegistrationForm = () => {
             <div className="mb-4 text-xl font-semibold">Steps</div>
             <Menu steps={steps} activeStep={activeStep} size="large" />
```

Comment:
let's remove the steps for now... we'll just have Speaker List (if they are available)